apps/spruce/src/pages/waterfall/WaterfallFilters.tsx

```diff
@@ -23,9 +23,9 @@ export const WaterfallFilters: React.FC<WaterfallFiltersProps> = ({
 
   return (
     <Container>
```

Comment:
While you're here, any chance you could also lessen the container padding like we discussed with Will?

Reply:
lmk if it's still too big!